gammapy/image/utils.py

```diff
@@ -121,11 +123,20 @@ def disk_correlate(image, radius, mode='constant'):
 
     Parameters
     ----------
-    TODO
+    image : array_like
+        Image to be correlated.
+
+    radius : float
+        Disk radius in pixels.
+
+    mode : {‘reflect’,’constant’,’nearest’,’mirror’, ‘wrap’}, optional
```

Comment:
Ok, it's strange (I just copied the strings from the scipy covolve doc). Thanks for the tip with building sphynx and the link to the local doc.